queries/nvim/highlights.scm

```diff
@@ -16,14 +16,47 @@
   "then"
   "when"
 ] @keyword
-["def" "fun"] @keyword.function
-["class" "enum" "module" "struct" "type" "union"] @keyword.type
-["require"] @keyword.import
-["return" "yield"] @keyword.return
-["if" "else" "elsif" "unless"] @keyword.conditional
-["while" "until"] @keyword.repeat
-["rescue"] @keyword.exception
-[(private) (protected) "abstract"] @keyword.modifier
+
+[
+  "def"
+  "fun"
+] @keyword.function
+
+[
+  "class"
+  "enum"
+  "module"
+  "struct"
+  "type"
+  "union"
+] @keyword.type
```

Comment:
Shouldn't this also include `lib`?

Reply:
As well, `type` is only a keyword within a `lib`, unless that's already captured by this

Reply:
I'll move lib here them.

About`type`, is works because `"type"` token is only emitted inside a lib node.